spec/factories/nca_facilities.rb

```diff
@@ -159,6 +163,7 @@
     facility_type  'va_cemetery'
     classification 'National Cemetery'
     website 'https://www.cem.va.gov/cems/nchp/cypresshills.asp'
+    mobile nil
```

Comment:
Is nil a default? Seems like maybe it should be in the factory? And that raises the question if it should be a default in the db.

Reply:
+1 to this. If we can have mobile default to `nil` in the db somehow then maybe we can avoid all these changes in the factories

Reply:
The DB does default to nil, conveniently! I was just trying to communicate intent by adding nil explicitly to these fixtures. But it seems like it causes more confusion than not including it at all. I can take them back out!